pkg/errors/errors.go

```diff
@@ -21,6 +21,7 @@ var (
 	ErrInvalidArgument = errors.Normalize("invalid argument", errors.RFCCodeText("BR:Common:ErrInvalidArgument"))
 	ErrVersionMismatch = errors.Normalize("version mismatch", errors.RFCCodeText("BR:Common:ErrVersionMismatch"))
 	ErrFailedToConnect = errors.Normalize("failed to make gRPC channels", errors.RFCCodeText("BR:Common:ErrFailedToConnect"))
+	ErrFailedToExecute = errors.Normalize("failed to execute SQL", errors.RFCCodeText("BR:Common:ErrFailedToExecute"))
```

Comment:
given that the SQL can be anything this error looks no better than ErrUnknown 🙃

Reply:
Removed 🤔